docs/models/extensions/runai_model_streamer.md

```diff
@@ -24,6 +24,13 @@ vllm serve s3://core-llm/Llama-3-8b \
     --load-format runai_streamer
 ```
 
+To run model from Google Cloud Storage run:
+
+```bash
+vllm serve gs://core-llm/Llama-3-8b \
+    --load-format runai_streamer
```

Comment:
From Google cloud storage load model, whether need set like s3 AWS_ACCESS_KEY_ID or AWS_SECRET_ACCESS_KEY, if yes, can we make this example better?